examples/profile-cards-graphql/src/main.js

```diff
@@ -27,7 +28,8 @@ const QUERY = gql`
   }
 `;
 
-const props = ({ data }) => data.loading ? { users: [] } : { users: data.allProfiles };
+const props = ({ data }) =>
+  (data.loading ? { users: [] } : { users: data.allProfiles });
```

Comment:
The bracket matter, the linter said the expression is confusing;  It could be the same line;